.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is added by mistake?
If yes, I think we need to remove this file.

Reply:
Sorry, that was added by mistake. I removed it now. Could you please check it again?